test/e2e/lib/hooks/browser/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Internal dependencies
+ */
+import { saveBrowserLogs } from './browser-logs';
+import { createBrowser, closeBrowser } from './browser';
+
+export const buildHooks = () => {
```

Comment:
Not a nitpick, but a question: is there a reason for sticking with the syntax 
`export const <name> = () => {}`
instead of
`export function <name>`?

Reply:
Force of habit mostly. I only use `function <name>` if the function uses `this` (which some hooks do in Mocha world), and `export const <name> = () => {}` for everything else.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions has a good list of things an arrow function expression can't do.